server/build/plugins/combine-assets-plugin.js

```diff
@@ -11,26 +11,19 @@ export default class CombineAssetsPlugin {
 
   apply (compiler) {
     compiler.plugin('compilation', (compilation) => {
-      compilation.plugin('additional-chunk-assets', (chunks) => {
```

Comment:
I think we've this as a separate PR?
So, you want to close that or remove these changes from here?

Reply:
Merged 👍